src/w3c/templates/headers.js

```diff
@@ -280,6 +270,22 @@ export default (conf, options) => {
   </div>`;
 };
 
+function renderSpecTitle(conf) {
+  const specType = conf.isCR ? `${conf.longStatus}` : `${conf.textStatus}`;
+  const preamble = conf.prependW3C
+    ? html`<a href="https://www.w3.org/standards/types">W3C ${specType}</a
```

Comment:
Is there a subpath or fragment for specific types?

Reply:
There is, but not for the "MVP"... will add that later.